frontend/src/components/dashboard/ProfileBanners.tsx

```diff
@@ -2,13 +2,7 @@ import { ReactNode } from "react";
 
 import styles from "./ProfileBanners.module.scss";
 import FirefoxLogo from "./images/fx-logo.svg";
-import BundleLogo from "./images/vpn-and-relay-logo.svg";
```

Comment:
change (blocking): looks like we can remove the `frontend/src/components/landing/images/vpn-and-relay-logo.svg` and `frontend/src/components/dashboard/images/vpn-and-relay-logo.svg` files too.